services/user-svc/internal/organization/organization.go

```diff
@@ -175,13 +176,65 @@ func (s ServiceServer) GetOrganization(ctx context.Context, req *pb.GetOrganizat
 }
 
 func (s ServiceServer) AddMember(ctx context.Context, req *pb.AddMemberRequest) (*pb.AddMemberResponse, error) {
-	// TODO
-	return nil, status.Errorf(codes.Unimplemented, "method AddMember not implemented")
+	log := zlog.Ctx(ctx)
+	db := hwgorm.GetDB(ctx)
+
+	userID, err := uuid.Parse(req.UserId)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	organizationID, err := uuid.Parse(req.Id)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	if err := AddUserToOrganization(ctx, db, userID, organizationID); err != nil {
+		return nil, err
+	}
+
+	// Make the creating user admin of the organization
+	if err := ChangeMembershipAdminStatus(db, userID, organizationID, false); err != nil {
+		return nil, err
+	}
```

Comment:
The default value of `IsAdmin` is already false. I guess we don't really need to do this twice.